nuget/bblanchon.PDFium.Win32.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>bblanchon.PDFium.Win32</id>
+    <version>$version$</version>
+    <authors>Benoit Blanchon</authors>
+    <owners>Benoit Blanchon</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
+    <projectUrl>https://github.com/bblanchon/pdfium-binaries</projectUrl>
+    <description>This package contains pre-compiled binaries of the PDFium library, an open-source library for PDF manipulation and rendering.</description>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <copyright>Copyright © Benoit Blanchon 2017-2022</copyright>
+    <tags>PDFium PDF binaries library native Chromium</tags>
+    <repository type="git" url="https://github.com/bblanchon/pdfium-binaries.git" branch="$branch$" commit="$commit$" />
+
+    <dependencies>
+      <group targetFramework="netstandard2.0" />
+    </dependencies>
+  </metadata>
+
+  <files>
+    <!-- placeholder file because this package contains native binaries only -->
+    <file src="_._" target="lib/netstandard2.0/_._" />
```

Comment:
Yes, it is. Otherwise you get these NuGet warning when packaging:

[`WARNING: NU5127: This package does not contain a lib/ or ref/ folder, and will therefore be treated as compatible for all frameworks. Since framework specific files were found under the build/ directory for net461, consider creating the following empty files to correctly narrow the compatibility of the package:
-lib/net461/_._`](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5127)

[`WARNING: NU5128: Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder do not have exact matches in the other location. Consult the list of actions below: - Add lib or ref assemblies for the netstandard2.0 target framework`](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128)

I can add the URL of the first warning as a comment though.